src/pages/StatisticsChart/activities/ContractResourceDistributed.tsx

```diff
@@ -0,0 +1,104 @@
+import { useTranslation } from 'react-i18next'
+import { DATA_ZOOM_CONFIG } from '../../../utils/chart'
+import { SmartChartPage } from '../common'
+import { ChartItem, explorerService } from '../../../services/ExplorerService'
+import { ChartColorConfig } from '../../../constants/common'
+import config from '../../../config'
+
+const useOption = (
+  statisticContractResourceDistributed: ChartItem.ContractResourceDistributed[],
+  chartColor: ChartColorConfig,
+  isMobile: boolean,
+  isThumbnail = false,
+): echarts.EChartOption => {
+  const { t } = useTranslation()
+  const gridThumbnail = {
+    left: '4%',
+    right: '10%',
+    top: '8%',
+    bottom: '6%',
+    containLabel: true,
+  }
+  const grid = {
+    left: '3%',
+    right: '3%',
+    top: '8%',
+    bottom: '5%',
+    containLabel: true,
+  }
+  return {
+    color: chartColor.colors,
+    grid: isThumbnail ? gridThumbnail : grid,
+    dataZoom: isThumbnail ? [] : DATA_ZOOM_CONFIG,
+    xAxis: [
+      {
+        name: isMobile || isThumbnail ? '' : t('statistic.address_count'),
+        nameLocation: 'middle',
+        nameGap: 30,
+        type: 'log',
+        splitLine: { show: false },
+      },
+    ],
+    yAxis: [
+      {
+        type: 'log',
+        splitLine: { show: false },
+        name: isMobile || isThumbnail ? '' : t('statistic.ckb_amount'),
+      },
+    ],
+    tooltip: {
+      position: 'top',
+      enterable: true,
+      formatter: (params: any) => {
+        return `<a href=https://${config.BASE_URL}script/${params.value[3]}/${params.value[5]} target=_blank>${
+          params.value[4] || params.value[3]
+        }</a>`
+      },
+    },
+    series: [
+      {
+        type: 'scatter',
+        symbol: 'circle',
+        symbolSize: (data: number[]) => {
+          const ratio = isThumbnail ? 500 : 50
+          const min = isThumbnail ? 1 : 10
+          const size = Math.sqrt(data[2]) / ratio
+          return size < min ? min : size
+        },
+      },
+    ],
+    dataset: {
+      source: statisticContractResourceDistributed
+        .filter(item => item.addressCount !== '0' || item.ckbAmount !== '0')
+        .map(data => [data.addressCount, data.ckbAmount, data.txCount, data.codeHash, data.name, data.hashType]),
+    },
+  }
+}
+
+const toCSV = (statisticContractResourceDistributed: ChartItem.ContractResourceDistributed[]) =>
+  statisticContractResourceDistributed
+    ? statisticContractResourceDistributed.map(data => [
+        data.name,
+        data.codeHash,
+        data.hashType,
+        data.txCount,
+        data.ckbAmount,
+        data.addressCount,
+      ])
+    : []
+
+export const ContractResourceDistributedChart = ({ isThumbnail = false }: { isThumbnail?: boolean }) => {
+  const [t] = useTranslation()
+  return (
+    <SmartChartPage
+      title={t('statistic.contract_resource_distributed')}
```

Comment:
The `description` field is expected to be added as
<img width="542" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/7271329/e56f304e-1f67-4f85-81cd-c74364531d84">